src/app/components/Statistics/CO2ConsumptionCharts/CountriesBarMeter.tsx

```diff
@@ -2,18 +2,21 @@ import React from 'react'
 import Highcharts from 'highcharts'
 import HighchartsReact from 'highcharts-react-official'
 import sortBy from 'lodash/sortBy'
+import { Data, IDataChart, Meter, Value } from 'types/statistics'
 
-const CountriesBarMeter = ({ darkMode, titleText, dataSeries }) => {
-    const sortByAndLoadBar = data => {
-        let sortedData = sortBy(data, o => -o.Value)
-        let dataChart = sortedData.map((dat, index) => {
+const CountriesBarMeter = ({ darkMode, titleText, dataSeries }: Meter) => {
+    const sortByAndLoadBar = (data: Value): Data[] => {
+        // @ts-ignore:next-line
+        let sortedData: Value[] = sortBy(data, object => object && -object.Value)
+        let dataChart = sortedData.map((dat, index): Data => {
+            const country = dat.Country || ''
```

Comment:
`dat` should be renamed to `data`

Reply:
Hi @Chriserus, we already changed the dat for data